view/frontend/web/images/alipay.svg

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please optimise SVGs. There's a lot of cruft in here that's not needed. This tool might help: https://jakearchibald.github.io/svgomg/

Reply:
Good shout!